packages/stacked_cli/example/lib/ui/setup/setup_bottom_sheet_ui.dart

```diff
@@ -1,18 +1,23 @@
 import 'package:flutter/material.dart';
 import 'package:example/app/app.locator.dart';
 import 'package:example/enums/bottom_sheet_type.dart';
-import 'package:example/ui/bottom_sheets/notice_sheet/notice_sheet.dart';
+import 'package:example/ui/bottom_sheets/notice/notice_sheet.dart';
+// @stacked-bottom-sheet-import
```

Comment:
We can use the existing template identifier for this `// @stacked-import`

Reply:
You are right, I don't know why I didn't think of that yesterday.